packages/web-client/app/components/card-space/edit-details/description/index.ts

```diff
@@ -0,0 +1,43 @@
+import Component from '@glimmer/component';
+import { WorkflowCardComponentArgs } from '@cardstack/web-client/models/workflow';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class CardSpaceEditDetailsDescriptionComponent extends Component<WorkflowCardComponentArgs> {
+  @tracked validationMessage: string | null = null;
+  @tracked description: string | null = null;
```

Comment:
This is quite minor but is there any value in using `null` for the types instead of `undefined`? Then it can be left as the default.